### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN049.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#98] [General]
# https://blog.3place.cn/3place/lmyn/LMYN049.html

https://blog.3place.cn/3place/lmyn/LMYN049.html

**Comment by picklecai** [comment]
他生气诅咒我失去生殖力，
他又诅咒她失去了形体。
他发出了这样激烈的诅咒，
他的苦行被我破坏冲击。 （1.48.3）

这两个诅咒，十分切中要害。

**Comment by picklecai** [comment]
这一位小心谨慎的女子，
献上洗脚水招待客人。
罗摩遵照应有的礼节，
也对她回敬献殷勤。 （1.48.18）

洗脚水，很重要。

**Comment by picklecai** [comment]
大阵花雨降碧空，
天鼓声声相和鸣，
乾闼婆们和天女，
成群结队来聚拢。 （1.48.19）

天女散花。
天花乱坠。
伴随的乾达婆，就是为了提供仙乐吧。

**Comment by picklecai** [comment]
罗摩解救阿诃厘耶，就好像孙悟空路过朱紫国时，顺便救一下金圣宫娘娘一样。

**Comment by picklecai** [comment]
听因陀罗把话说完，
便同所有的摩噜陀群，
走近祖先和神仙开了言： （1.48.5）

注释说摩噜陀是因陀罗的伙伴，也就是上文底提的子宫被分裂出来的七块（七个风神）。
